Documentation/Compass/README.md

```diff
@@ -0,0 +1,25 @@
+# Compass
+
+The Compass (alias North arrow) shows where north is in the MapView. The Compass upports reset rotation.
+
+The ArcGIS Runtime SDK currently supports rotating the map with 2-finger gesture on MapView and SceneView interactively by default but there is no way of re-orientating to north without developers programming it. This leaves the map in a difficult state to re-orientate back to north. The Compass Toolkit component will appear when the map is rotated and, when tapped, re-orientates the map back to north and hides the compass icon (note that the MapView auto-snaps back to north when it's within a threshold of north, and in that case the compass also auto hides).
```

Comment:
> but there is no way of re-orientating to north without developers programming it

This isn't true. The map will snap to north when rotating with 2 fingers. The compass just provides an easier way.